documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -13,13 +13,24 @@ For an overview of the `Kafka` resource, see the xref:ref-sample-kafka-resource-
 You can configure `cruiseControl` properties as part of a deployment or redeployment of a Kafka cluster.
 Deploy one instance of Cruise Control per Kafka cluster.
 
+In this procedure, we use the example file provided with {ProductName}:
+
+* `examples/topic/cruise-control-topics.yaml`
```

Comment:
_I think you need a sentence or two on the purpose of the topics file. Also, "we use" sounds like this method of deploying Cruise Control is optional._ 

_Something like this might work:_

This procedure uses the following example file provided with {ProductName}:

`examples/topic/cruise-control-topics.yaml`

This creates a  `strimzi.cruisecontrol.metrics` topic in your Kafka cluster, which collects information from the Cruise Control Metrics Reporter for use in generating rebalance proposals. You do not need to interact with this topic after Cruise Control is deployed.